### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by harisma-git**
Hi... I made some changes to miio so that the cleaning time and the area appear properly.

### CODE REVIEW

File: miio/dreamevacuum.py [hunk 1]
```diff
@@ -139,9 +139,9 @@ class DreameStatus:
         default=None
     )
     # piid: 4 (area): (string, unit: None) (acc: ['read', 'write'], value-list: [], value-range: None)
-    area: str = field(metadata={"siid": 18, "piid": 4, "access": ["read", "write"]},default=None)
+    area: str = field(metadata={"siid": 18, "piid": 3, "access": ["read", "write"]},default=None)
```
Comment (DavidConnack) [inline miio/dreamevacuum.py:142]: What is piid with a value of 3?

File: miio/dreamevacuum.py [hunk 2]
```diff
@@ -139,9 +139,9 @@ class DreameStatus:
...
     # piid: 5 (timer): (string, unit: None) (acc: ['read', 'write'], value-list: [], value-range: None)
-    timer: str = field(metadata={"siid": 18, "piid": 5, "access": ["read", "write"]},default=None)
+    timer: str = field(metadata={"siid": 18, "piid": 2, "access": ["read", "write"]},default=None)
```
Comment (DavidConnack) [inline miio/dreamevacuum.py:144]: What is piid with a value of 2?

**Comment by DavidConnack** [comment]
Thanks

Please ensure to remove the .pyc files.
Also please confirm that you have tested and resolve all conflicts

**Comment by harisma-git** [comment]
The new piids in area and time are the correct ones. The old piids doenst work (at least on my vacuum) and thats why the vaccum card didnt show them correctly. Keep in mind that all this is for your cleaning zone "Version" . I dont know github very well so my pull request probably is wrong :/

**Comment by harisma-git** [comment]
The conflicts are probably as i have your old directory stracture in my repo. Deny the pull and i will make another.

**Comment by DavidConnack** [comment]
Yeah, it looks like an old version.
If you could pull the latest one.
You can install github desktop if you prefer to use a gui for github.

Thanks

**Pull request closed without merging**